gecoscc/views/report_printers.py

```diff
@@ -64,34 +65,32 @@ def report_printers(context, request, file_ext):
         report_type    : Type of report (html, csv or pdf)
     '''    
 
-    # Check current user permissions    
+    # Check current user permissions
     is_superuser = request.user.get('is_superuser', False)
-    ou = None 
-    
-    if not is_superuser:
-        # Get managed ous
-        ou_id = request.GET.get('ou_id', None)
-        if ou_id is None:
-            raise HTTPBadRequest()
-        
-        ou_visibles = request.user.get('ou_managed', []) + request.user.get('ou_readonly', [])
-        for oid in ou_visibles:
-            if oid == ou_id:
-                ou = ou_id
+
+    # Get managed ous
```

Comment:
This piece of code is repeated four times. Consider to move it to a common function/method

Reply:
Done.